fix: replace rgba() with rgb() in code blocks (#35613)
mdn/content: files/en-us/web/css/color_value/hsl/index.md

{{CSSRef}}

> [!NOTE]

> > The `hsla()` functional notation is an alias for `hsl()`. They both work the same. It is recommend to use `hsl()`.

Reviewer: ```suggestion
> The `hsla()` functional notation is an alias for `hsl()`. They are exactly equivalent. It is recommend to use `hsl()`.
```

Author: ```suggestion
> The `hsla()` functional notation is an alias for `hsl()`. They are exactly equivalent. It is recommended to use `hsl()`.
```

> > The `hsla()` functional notation is an alias for `hsl()`. They are exactly equivalent. It is recommended to use `hsl()`.

Reviewer: I think this can be omitted—it doesn't have to be _this_ visible. (Same for `rgba` obviously)

Author: The `hsla()` page url redirects to this page so the people looking for hsla() information need to know right away what happened to it.

hsl(from green h s l / 0.5)
hsl(from #0000FF h s calc(l + 20))
hsl(from rgb(200 0 0) calc(h + 30) s calc(l + 30))

/* Legacy 'hsla()' alias */
hsla(120deg 75% 25% / 60%)

/* Legacy format */
hsl(120, 75%, 25%)

> hsla(120deg, 75%, 25%, 0.8)

Reviewer: ```suggestion
hsl(120deg, 75%, 25%, 0.8)
```

Since we already established that hsla is an alias, let's just use `hsl`

Author: We've covered this case already on line 28 and extra hsla sample is no biggie. :)

Reviewer: Line 28 is the comma-less syntax. I'm saying here we should consistently demonstrate comma-list syntax using `hsl` because `hsla` is just exactly equivalent, and it's better to be consistent where variation is not the point.

hsl(from rgb(200 0 0) calc(h + 30) s calc(l + 30))
```

The `hsla()` function can also be used to express sRGB colors. This is an alias for `hsl()` that accepts the same parameters.

> > **Note:** `hsl()`/`hsla()` can also be written in a legacy form in which all values are separated with commas, for example `hsl(120, 75%, 25%)` or `hsla(120deg, 75%, 25%, 0.8)`. The `none` value is not permitted in the comma-separated legacy syntax, the `deg` on the hue value is optional, and the `%` units are required for the saturation and lightness values.

Reviewer: This needs to be either kept or be documented in the syntax box.

Author: done

{{EmbedLiveSample("legacy_syntax_comma-separated_values", "100%", 150)}}

> ### Legacy versus modern syntax

Reviewer: This needs to be kept, otherwise no one really knows what the legacy syntax looks like or what exactly it could do.

Author: I've added `hsla()` and legacy format to the to the syntax section. There is no need to show different formats that produce the same output in an example. :thinking:

Author: done. The example is back.
